recipes/pollster/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pollster" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: be83cceb4d6d9c8d9c2e31f21ddb360e29b6dfeff1948cba6f0ec0a3a2d7bfb4 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - certifi>=14.05.14
+    - six>=1.8.0
+    - python-dateutil>=2.5.3
+    - urllib3>=1.15.1
+    - pandas>=0.19.1
+
+test:
+  imports:
+    - pollster
+
+about:
+  home: https://elections.huffingtonpost.com/pollster
+  license: BSD-2-Clause
+  license_family: BSD
+  # License file is not packaged in the pypi tarball
+  # license_file: LICENSE
```

Comment:
Note: opened issue about this. https://github.com/huffpostdata/python-pollster/issues/15